### PR TITLE
fix: convert zip code from int to string for leading zero

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/7757

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by mariobehling** [comment]
Thanks!

**Comment by mariobehling** [comment]
@maze-runnar Circle-CI tests are failing.

File: migrations/versions/rev-2021-09-04-21:16:18-09c177e36ac2_.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+"""empty message
+
+Revision ID: 09c177e36ac2
```
Comment (progmatic-99) [inline migrations/versions/rev-2021-09-04-21:16:18-09c177e36ac2_.py:3]: You need to merge the different migration heads.

**Comment by maze-runnar** [comment]
> @maze-runnar Circle-CI tests are failing.

Fixed

**Review by ranjitkshah — APPROVED** [review]

**Comment by mariobehling** [comment]
This error still persists specifically in the "Company Information" widget on the right-hand side of orders.

Compare event https://eventyay.com/e/e02cbf16

Screenshot from 2021-09-23 17-50-44

**Comment by maze-runnar** [comment]
This should be a previously created event. For new events after that pr it's working.

**Pull request merged**